pkg/updater/gcs.go

```diff
@@ -390,12 +391,17 @@ func convertResult(log logrus.FieldLogger, nameCfg nameConfig, id string, header
 
 	emailAddressesInterface, ok := result.finished.Finished.Metadata[EmailListKey]
 	if ok {
-		emailAddresses, ok := emailAddressesInterface.([]string)
-		if ok {
-			out.Column.EmailAddresses = emailAddresses
-		} else {
-			log.Error("email addresses type is not list of strings")
+		// we do marshal and unmarshal to json to convert the result from []interface{} to []string
+		var emailAddresses []string
+		emailAddressesData, err := json.Marshal(emailAddressesInterface)
```

Comment:
This is a horribly inefficient way to do this.

Reply:
The email list is supposed to be amount of committers, not should be  a lot of  them.
But I had another method here that is more efficient but also more complex

https://github.com/GoogleCloudPlatform/testgrid/pull/602/files/cde6941ddad425290e448cfe83672ff3494f5468